share/build/buildlib.ekat

```diff
@@ -113,9 +128,13 @@ def buildlib(bldroot, installpath, case):
     gmake_j = case.get_value("GMAKE_J")
 
     # Just enable all ekat packages?
-    ekat_cmake_options = "-DEKAT_ENABLE_ALL_PACKAGES=On -DEKAT_ENABLE_TESTS=Off"
+    debug = case.get_value("DEBUG")
+    ekat_cmake_options = "-DEKAT_ENABLE_ALL_PACKAGES=On -DEKAT_ENABLE_TESTS=Off -DKokkos_ENABLE_SERIAL=On"
     if mpilib == "mpi-serial":
         ekat_cmake_options += " -DEKAT_ENABLE_MPI=Off"
+    if debug:
+        ekat_cmake_options += " -DCMAKE_BUILD_TYPE=Debug"
+        # More? Turn on kokkos debugging?
```

Comment:
Do we need more kokkos settings when DEBUG is true? Like KOKKOS_DEBUG or bounds checking?

Reply:
KOKKOS_DEBUG is defaulted to ON for build type is DEBUG (case insensitive), and OFF otherwise. I would not automatically set bounds check to ON in debug. It significantly increases the runtime, IIRC.